libs/rtefsutils/include/RteFsUtils.h

```diff
@@ -248,9 +248,8 @@ class RteFsUtils
    * @brief normalize path by ensuring that it's absolute and canonical
    * @param path folder to be normalized
    * @param base root folder to form absolute
-   * @return true if path is normalized
   */
-  static bool NormalizePath(std::string& path, std::string base = "");
+  static void NormalizePath(std::string& path, std::string base = "");
```

Comment:
This PR depends on this change, since evaluation will fail if the normalized paths don't exist when the RTE files not have been created. I can put this in a separate PR if you like, but then this PR will depend on that one.

Reply:
New PR with this change submitted as #436